Formula/aws-okta.rb

```diff
@@ -11,6 +11,8 @@ class AwsOkta < Formula
     sha256 cellar: :any_skip_relocation, catalina:      "2edc4ebb817ff4f0a3188a0c0eea6416ce2a83a6d9b5cc5b3969034ee65e27ca"
     sha256 cellar: :any_skip_relocation, mojave:        "910418c2dd89b78a7d665cdd8082d9941de433c6c8db800ce0515dfb6c1eb25b"
   end
+  # see https://github.com/segmentio/aws-okta/issues/278
```

Comment:
Minor suggestion:
```suggestion

  # See https://github.com/segmentio/aws-okta/issues/278
```
Also, please squash all your commits into one commit, with the message:
```
aws-okta: deprecate
```

Reply:
sure thing

Reply:
done